src/DataSource.ts

```diff
@@ -149,7 +159,63 @@ export class DataSource extends DataSourceApi<GrafanaQuery, GenericOptions> {
     );
   }
 
-  listMetrics(filter: string, options?: any, type?: string): Promise<MetricFindValue[]> {
+  listMetricPayloadOptions(name: string, metric: string, payload: string | { [key: string]: string }): Promise<Array<SelectableValue<string | number>>> {
+    return lastValueFrom<Array<SelectableValue<string | number>>>(
+      this.doFetch({
+        url: `${this.url}/options`,
+        data: {
+          metric,
+          payload: this.processPayload(payload, "builder", undefined),
+          name
+        },
+        method: 'POST',
+      }).pipe(
+        map((response) => isArray(response.data) ? response.data.map((item) => ({ ...item, label: item.label ?? item.value })) : []),
+
+        catchError((err) => {
+          global.console.error(err);
```

Comment:
Why is somewhere `global.console.error?` and somewhere console.error?

Reply:
When I debug locally, console.log cannot work normally. I can only use global.console instead of console

Reply:
Maybe you have it somehow overridden 🤷🏾‍♀️ Anyway, let's keep standard one for the contribution.

```suggestion
         console.error(err);
```